datastore/db/metadata/inspection/__init__.py

```diff
@@ -41,32 +46,29 @@ class OrganizationInformation(BaseModel):
 
 
 class Value(BaseModel):
-    value: Optional[float] = None
-    unit: Optional[str] = None
-    name: Optional[str] = None
-    edited: Optional[bool] = False
+    value: float | None = None
+    unit: str | None = None
+    edited: bool = False
 
 
-class ValuesObjects(BaseModel):
-    en: List[Value] = []
-    fr: List[Value] = []
+class NamedValue(Value):
+    name: str | None = None
 
 
-class SubLabel(BaseModel):
-    en: List[str] = []
-    fr: List[str] = []
+class LocalizedNamedValue(BaseModel):
+    en: list[NamedValue] = []
+    fr: list[NamedValue] = []
 
 
-class Metric(BaseModel):
-    value: Optional[float] = None
-    unit: Optional[str] = None
-    edited: Optional[bool] = False
+class SubLabel(BaseModel):
+    en: list[str] = []
+    fr: list[str] = []
 
 
 class Metrics(BaseModel):
-    weight: Optional[List[Metric]] = []
-    volume: Optional[Metric] = Metric()
-    density: Optional[Metric] = Metric()
+    weight: list[Value] = []
```

Comment:
I liked the idea of keeping a metric and value different (even if the current structure is the same). In the future, we might of added more fields in each of them since they are fundamentally different.

For the moment this changes has no impact.

Reply:
Fair point. But removing code smells (duplication in this case) is almost never bad.